examples/Example.h

```diff
@@ -376,7 +376,8 @@ struct WgWindow : Window
     {
         if (!initialized) return;
 
-        window = SDL_CreateWindow("ThorVG Example (WebGPU)", SDL_WINDOWPOS_UNDEFINED, SDL_WINDOWPOS_UNDEFINED, width, height, SDL_WINDOW_HIDDEN);
+        // since wgpu_native v22.x.x window must be shown for a proper context configuration
```

Comment:
Please remove the comment here, you can add the history in the commit msg instead.